scripts/.storybook/manager.js

```diff
@@ -1,5 +1,6 @@
 import { addons } from '@storybook/addons';
 import contentfulTheme from './contentful-theme';
+import '!style-loader!css-loader!./osano.css';
```

Comment:
I found this solution in [this disussion](https://github.com/storybookjs/storybook/issues/6364). This is the only way we were able to make it work 🤷‍♀️